### ISSUE TITLE
Installation Errors

### ISSUE THREAD

**Issue by hadasara** [SuperElastix/SimpleElastix#196]
Hi all,
I'm tryin to build simpleElastix by following the instructions here:
http://simpleelastix.readthedocs.io/GettingStarted.html#compiling-on-windows 
but I get stuch in the very begining, with CMake unable to build the project. I get the error:

"Error in configuration process, project files may be invalid"

it is strange, in the log file- it says 0 Error(s) everywere... (log atached)

I have Visual Studio Community 2017 64bit, I use CMake 3.11.2 (win64-x64 Installer)
I have Windows 7 64bit.
I downloded the source files from: https://github.com/SuperElastix/SimpleElastix and save them on my computer.

I'd be happy to get some help.
Thanks!

[CMakeOutput.log](https://github.com/SuperElastix/SimpleElastix/files/2027591/CMakeOutput.log)

**Comment by kaspermarstal** [comment]
Can you show the log from the configuration step in which the error occurs?

**Comment by hadasara** [comment]
So, I don't know exactly what I did different, but now CMAke succeeded.
I used VS 2013 now, 
and I pressed config twice, firsst time it was red, second time white.
I moved forward but I think visual studio didn't suceed to build the project. I got many (72) warnings like this about ITK project:

`Warning	1	warning LNK4221: This object file does not define any previously undefined public symbols, so it will not be used by any link operation that consumes this library [E:\SpElx\SpEx_m\build\ITK-build\Modules\Core\Common\src\ITKCommon.vcxproj]	E:\SpElx\SpEx_m\build\itkAtomicInt.obj	ITK

and 2 errors:`

`Error	73	error MSB6006: "cmd.exe" exited with code 1.	C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppCommon.targets	170	5	SimpleITK
Error	74	error MSB6006: "cmd.exe" exited with code 1.	C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppCommon.targets	170	5	SimpleITKExamples
`
I attach the CMakeOUtput log file that was generated:
[CMakeOutput.log](https://github.com/SuperElastix/SimpleElastix/files/2045964/CMakeOutput.log)

Does anyone has an idea?

Thanks!
